### PR TITLE
[T4A5][T13-B1] Nathan Ohun Kwon 

### PR DESCRIPTION
**Pull request by kwonn**
Ready for review.

### CODE REVIEW

File: src/seedu/addressbook/data/tag/Tagging.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+import seedu.addressbook.data.person.Person;
+
+public class Tagging {
+
+    private final String tagAction;
+    private final Person person;
+    private final Tag tag;
+    
+    public static final String ADD_TAG = "+";
```
Comment (okkhoy) [inline src/seedu/addressbook/data/tag/Tagging.java]: think why `String` may not be the best data type for this particular field? 
what would be more appropriate?

File: src/seedu/addressbook/data/tag/Tagging.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.Person;
+
+public class Tagging {
+
+    private final String tagAction;
```
Comment (okkhoy) [inline src/seedu/addressbook/data/tag/Tagging.java:7]: think why `String` may not be the best data type for this particular field? 
what would be more appropriate?

File: src/seedu/addressbook/data/tag/Tagging.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    public Tagging(String tagAction, Person person, Tag tag) {
+        this.person = person;
+        this.tag = tag;
+        this.tagAction = tagAction;
+    }
+    
+    public String toString() {
+        if (tagAction == "ADD") {
+            return ADD_TAG + " " + person.getName().toString() + " " + tag.toString(); 
+        }
```
Comment (okkhoy) [inline src/seedu/addressbook/data/tag/Tagging.java:23]: coding standard violation :-1: 
`else` should be cuddled between braces.

**Comment by okkhoy** [comment]
@kwonn 
Some comments added. Please ack & close the PR after reading comments.

File: src/seedu/addressbook/data/tag/Tagging.java [hunk 4]
```diff
@@ -0,0 +1,28 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.Person;
+
+public class Tagging {
+
+    private final String tagAction;
+    private final Person person;
+    private final Tag tag;
+
```
Comment (okkhoy) [inline src/seedu/addressbook/data/tag/Tagging.java:10]: where is your tagging list maintained?

File: src/seedu/addressbook/data/tag/Tagging.java [hunk 5]
```diff
@@ -0,0 +1,28 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.Person;
+
```
Comment (okkhoy) [inline src/seedu/addressbook/data/tag/Tagging.java:4]: Please include a header comment to indicate what this interface/class/method does?

**Comment by kwonn** [comment]
Thanks!

**Pull request closed without merging**